### PR TITLE
Changes in index.ts to remove usage of environment variable INFURA_TOKEN

### PR DESCRIPTION
**Pull request by Blizzard57**
## Solves

It makes the front end work with any RPC node directly with the URL provided, rather than with an INFURA_TOKEN which was required initially

## How to test it

The react app will open when doing `yarn start`

### CODE REVIEW

**Review by 0xtemujin — CHANGES_REQUESTED** [review]
Pls add the RPC endpoint to .env and import that where needed

File: src/utils/constants.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const BEAMER_ID = IS_PRODUCTION ? process.env.REACT_APP_BEAMER_ID : 'ehlR
 export const SENTRY_DSN = process.env.REACT_APP_SENTRY_DSN || ''
 export const PORTIS_ID = process.env.REACT_APP_PORTIS_ID ?? '852b763d-f28b-4463-80cb-846d7ec5806b'
 export const FORTMATIC_KEY = process.env.REACT_APP_FORTMATIC_KEY ?? 'pk_test_CAD437AA29BE0A40'
-export const INFURA_TOKEN = process.env.REACT_APP_INFURA_TOKEN || ''
+export const RPC_ENDPOINT = process.env.REACT_APP_PRC_NODE || ''
```
Comment (zjesko) [inline src/utils/constants.ts:16]: nit: typo

File: src/utils/constants.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const BEAMER_ID = IS_PRODUCTION ? process.env.REACT_APP_BEAMER_ID : 'ehlR
...
 export const SAFE_APPS_RPC_TOKEN = process.env.REACT_APP_SAFE_APPS_RPC_INFURA_TOKEN || ''
```
Comment (zjesko) [inline src/utils/constants.ts]: please accommodate for this INFURA_TOKEN as well

**Review by zjesko — CHANGES_REQUESTED** [review]

File: src/utils/constants.ts (hunk 1 above)
Comment (zjesko) [reply src/utils/constants.ts:16]: Also, could you add this variable to .env.example?

File: src/utils/constants.ts (hunk 2 above)
Comment (Blizzard57) [reply src/utils/constants.ts]: I have removed the other instance of INFURA_TOKEN as well. Please take a look at it.

**Pull request closed without merging**